lib/themes/dosomething/paraneue_dosomething/js/reportback/Reportback.js

```diff
@@ -308,7 +327,7 @@ const Reportback = {
         'isListItem': true,  // @TODO: probably a better way to address this... :insert shruggy emoji here:
         'reactions': {
           enabled: setting('dsKudosReactions.enabled'),
-          data: this.getReactions(items[i].kudos.data),
+          data: setting('dsKudosReactions.likedByCurrentUser') ? this.getReactions(items[i].kudos, true) : this.getReactions(items[i].kudos.data, false),
         }
```

Comment:
@DFurnes START HERE! if it is not coming from Rogue, each item has `kudos.data`. However, this is not how it is returned in the Rogue response. So I manipulated it here but again, I don't think this is the correct way to go about this? I didn't change the response in the Rogue data because it won't ever send multiple reactions, just one.